### PR TITLE
maxrichmond: list bug

### PR DESCRIPTION
**Pull request by maxrichie5**
### Background

I accidentally introduced a bug that caused Dicts to be considered lists

### Changes

* checking isinstance instead of hasattr because dicts are iterable
* added unit test that ensures two version spots are equal

### Testing

* Tested on local repo that was breaking before this change

### CODE REVIEW

**Review by darwayne — DISMISSED** [review]

**Review by darwayne — COMMENTED** [review]
Can we also get a test in so we don't accidentally introduce a regression for this in the future?

**Comment by maxrichie5** [comment]
> Can we also get a test in so we don't accidentally introduce a regression for this in the future?

Good idea

File: panther_analysis_tool/util.py [hunk 1]
```diff
@@ -138,4 +138,4 @@ def deep_get(obj: Dict, path: List[str], default: Any = None) -> Any:
 
 
 def to_list(listish: Any) -> List:
-    return listish if hasattr(listish, "__iter__") else [listish]
+    return listish if isinstance(listish, list) else [listish]
```
Comment (cdzombak) [inline panther_analysis_tool/util.py:141]: @maxrichie5 does this need to handle other list-ish things (`set`, `tuple`)?

Comment (cdzombak) [reply panther_analysis_tool/util.py:141]: (that'd be `isinstance(listish, tuple)` and `isinstance(listish, set)` IIRC), but of course tests should cover those too.

Comment (maxrichie5) [reply panther_analysis_tool/util.py:141]: I can add those to the tests. It should not. It is only supposed to make a single a list and keep a list a list

Comment (cdzombak) [reply panther_analysis_tool/util.py:142]: Gotcha. I'd like to see a test to document that behavior explicitly, but I'll provisionally approve the PR — just merge once you have that done, please.

**Review by cdzombak — DISMISSED** [review]

File: panther_analysis_tool/util.py (hunk 1 above)
Comment (cdzombak) [reply panther_analysis_tool/util.py:142]: @maxrichie5 yes! :shipit:

**Pull request merged**